prompt2model/dataset_retriever/column_selection_prompt.py

```diff
@@ -0,0 +1,95 @@
+        This dataset has the following columns: [sentences, questions, answers, options]""",  # noqa: E501
+        """{
+        "input": ["context", "question"],
+        "output": ["answer"],
+        "irrelevant": [],
+        "ambiguous": ["options"]
+        }""",
+    ),
+    (
+        """Your job is to be able to translate between languages. You will use the Opus100 dataset from HuggingFace. A sample data instance from this is as follows:  {"translation":{ "ca": "El department de bombers té el seu propi equip d'investigació.", "en": "Well, the fire department has its own investigative unit." }}. This dataset has the following columns: [translation]. """,  # noqa: E501
+        """{
+        "input": [],
+        "output": [],
+        "irrelevant": []
+        "ambiguous": ["translation"]
+        }""",
+    ),
+]
+
+
+def truncate_row(example_row: dict) -> str:
+    """Add truncation if the example row is too big."""
+    truncated_row = {}
+    for key in example_row.keys():
+        truncated_row[key] = json.dumps(example_row[key])[:50]
+    return json.dumps(truncated_row)
```

Comment:
The length at which truncation occurs should be a parameter.
```suggestion
def truncate_row(example_row: dict, max_length: int = 50) -> str:
    """Add truncation if the example row is too big."""
    truncated_row = {}
    for key in example_row.keys():
        truncated_row[key] = json.dumps(example_row[key])[:max_length]
    return json.dumps(truncated_row)
```

Also, we may want to indicate that it was truncated by adding "..." or something?